exonum/src/blockchain/mod.rs

```diff
@@ -307,7 +307,7 @@ impl BlockchainMut {
         }
 
         // Save & execute transactions.
-        for (index, hash) in tx_hashes.iter().enumerate() {
+        for (index, hash) in (0..).zip(tx_hashes.iter()) {
```

Comment:
Nit: Isn't `iter()` redundant here? That is, `(0..).zip(tx_hashes)` would work as well.